docs/source/rest-api.rst

```diff
@@ -1590,12 +1586,11 @@ Request Structure
 Get Latest ModelVersions
 ========================
 
-.. warning:: Model Stages are deprecated and will be removed in a future major release. To learn more about this deprecation, see our :ref:`migration guide<migrating-from-stages>`.
 
 +------------------------------------------------------+-------------+
 |                       Endpoint                       | HTTP Method |
 +======================================================+=============+
-| ``2.0/mlflow/registered-models/get-latest-versions`` | ``GET``     |
+| ``2.0/mlflow/registered-models/get-latest-versions`` | ``POST``    |
```

Comment:
The build-doc script generates `POST` . How to fix the build-doc script ?

Reply:
Let's compare to another endpoint that exposes both GET and POST (if it exists) and verify if there is some issue with the naming conventions with how this API is defined or what the logic needs to be get it to generate the appropriate docstrings.